handlebars/handlebars.go

```diff
@@ -178,9 +181,6 @@ func (e *Engine) Load() (err error) {
 // Execute will render the template by name
 func (e *Engine) Render(out io.Writer, template string, binding interface{}, layout ...string) error {
 	if !e.loaded || e.reload {
-		if e.reload {
```

Comment:
think this is the reload feature, if we remove this place we break the feature with it

Reply:
I couldn't find anywhere that `loaded` is evaluated in this package.  This code was added in a commit to fix an issue in an other package with regards to that engines `extend` function.  The reload feature works in testing without setting `loaded` to false.

I will leave it here as it's not necessary for the new fix but it is worth reviewing this logic next time there's a rewrite.